src/classes/UTIL_RecordTypes.cls

```diff
@@ -107,11 +107,7 @@ public class UTIL_RecordTypes {
         Map<Id, Schema.RecordTypeInfo> rtiById = objectType.getDescribe().getRecordTypeInfosById();
         if (rtiById.containsKey(recordTypeId)) {
             Schema.RecordTypeInfo rti = rtiById.get(recordTypeId);
-            if (!rti.isMaster() && rti.isAvailable()) {
```

Comment:
No, I don't think so.

If you look at [the prior implementation of UTIL_RecordTypes](https://github.com/SalesforceFoundation/Cumulus/blob/33d9a7ea14583c151d2ca93721c37e0092146c31/src/classes/UTIL_RecordTypes.cls#L44), the `getRecordTypeName` method was inspecting the `recordTypesById` map and returning the value from there (if it existed).  Taking out these access/master checks now makes the code follow essentially the same path as before: `sObjectType.getDescribe().getRecordTypeInfosById()` followed by `containsKey(recordTypeId)`, then returning the record type name.

My error in the rewrite was that I didn't notice that `getRecordTypeName` was using the `recordTypeIds` list (which is not filtered), instead of the `recordTypesList`, which is the only one that is filtered (for master record types and available record types).

Not only that, but I am pretty sure that the master record type isn't even returned by these `getRecordTypeInfos` methods.